raster/r.patch/support.c

```diff
@@ -1,6 +1,39 @@
 #include <grass/gis.h>
 #include <grass/raster.h>
 
+/**
+ * merges the statf[] arrays for each thread into
+ * Thread 0 for use in the support computation.
+ *
+ * the funciton takes in a 2D array thread_statf[][], where
+ * the rows are for represent each thread and each column is
+ * a file that is being used to patch. The end result is
+ * that thread_statf[0] is an array of size [nfiles] that holds
+ * the merged stats information.
+ *
```

Comment:
Please reread for typos. Start with uppercase as a sentence.

Reply:
Sorry, that was a careless mistake and has been corrected.

Reply:
No worries, thanks for fixing that.